#!/usr/bin/env python
PACKAGE = "controls"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("K_pos_T", double_t, 0, "K_pos_T", 0.1,  0, 1.5)
gen.add("K_vel_T", double_t, 0, "K_vel_T", 0.1,  0, 1.5)
gen.add("K_ang_T", double_t, 0, "K_ang_T", 0.1,  0, 1.5)
gen.add("K_angv_T", double_t, 0, "K_angv_T", 0.1,  0, 1.5)
gen.add("K_pos_C", double_t, 0, "K_pos_C", 0.1,  0, 1.5)
gen.add("K_vel_C", double_t, 0, "K_vel_C", 0.1,  0, 1.5)
gen.add("K_ang_C", double_t, 0, "K_ang_C", 0.1,  0, 1.5)
gen.add("K_angv_C", double_t, 0, "K_angv_C", 0.1,  0, 1.5)
gen.add("K_des_C", double_t, 0, "K_des_C", 1,  0, 1)

gen.add("delta_x", double_t,0, "delta_x", 3, 2 ,4)
gen.add("delta_y", double_t,0, "delta_y", 3, 2 ,4)
gen.add("delta_z", double_t,0, "delta_x", 0, -1 ,3)
gen.add("delta_yaw", double_t,0, "delta_y", 0, -1.5 ,1.5)

exit(gen.generate(PACKAGE, "formation", "formation"))

